vagrant-spk

```diff
@@ -705,24 +739,26 @@ def main():
         ('publish', publish,
             "Send the current packed app spk to \n"
             "  the Sandstorm App Market."),
-        ('halt', halt,
-            "Shut down the Vagrant VM."),
-        ('destroy', destroy,
-            "Delete the Vagrant VM (but leave \n"
+        ('vm', vm_subcommand,
+            "Pass through all following arguments to `vagrant`."),
+        ('halt', deprecated_halt,
+            "(deprecated, use 'vm halt') Shut down the Vagrant VM."),
+        ('destroy', deprecated_destroy,
+            "(deprecated, use 'vm destroy') Delete the Vagrant VM (but leave \n"
             "  --work-directory untouched)."),
-        ('global-status', global_status,
-            "Show the state of all active \n"
+        ('global-status', deprecated_global_status,
+            "(deprecated, use 'vm global-status') Show the state of all active \n"
```

Comment:
suggest: s/vm global-status/vagrant-spk vm global-status/

Rationale: Otherwise, people might think "vm" is a command available on their $PATH.

Same suggestion elsewhere.